constraints/src/main/java/gov/nasa/jpl/aerie/constraints/tree/ListExpressionAt.java

```diff
@@ -22,9 +22,6 @@ public DiscreteProfile evaluate(final SimulationResults results, final Interval
     //Even if the expression returns a discrete profile, this profile has a constant value, a list of serialized values,
     //at the singleton timepoint it has been computed at.
     //The instantiation and use of valueAt is necessary here because the profile are not only discrete but also "continuous" in the form of linear profiles.
-    if(!bounds.isSingleton()){
-      throw new Error("The ListExpressionAt node should be used only with singleton bounds.");
-    }
     final var retList = new ArrayList<SerializedValue>();
     for (final var field: elements) {
       retList.add(field.evaluate(results, bounds, environment)
```

Comment:
My concern here is that it is still being evaluated at `bounds.start`, and you've changed the bounds to `Interval.FOREVER`. Previously I used the bounds to tell these nodes when to evaluated the profiles, but now I think we need a separate argument so we don't evaluate everything at `Duration.MIN`.